formula/src/main/java/com/instacart/formula/internal/ManagerDelegate.kt

```diff
@@ -9,4 +9,8 @@ interface ManagerDelegate {
         result: Transition.Result<*>,
         evaluate: Boolean,
     )
+
+    fun onPostponedTransition(
```

Comment:
You've kinda alluded to it in the PR description, but can we document onPostponedTransition vs onTransition?

Reply:
I've refactored `ManagerDelegate` to contain only `onUpdate`